src/inabox/amp-inabox-lite.js

```diff
@@ -82,7 +83,9 @@ installPerformanceService(self); // TODO: to be removed
 
 self.document.documentElement.classList.add('i-amphtml-inabox');
 const fullCss =
-  cssText + 'html.i-amphtml-inabox{width:100%!important;height:100%!important}';
+  ampDocCss +
```

Comment:
Hmm. So why are we using `ampDocCss` here now?

Reply:
will have a follow up PR to run the change as experiment. keep logic exactly identical for now.

Reply:
Ok. Got it. Maybe let's leave a TODO.

Reply:
TODO added